### PR TITLE
Feat/style

### PR DESCRIPTION
**Pull request by 81jumpstr**
I expect a lot of merge conflicts...

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -16,8 +16,8 @@
 
   <body <?php body_class(); ?>>
     
-    <section id='header'>
+    <header class="site-header">
```
Comment (royscheeren) [inline header.php:19]: Nice. (Single quote should suffice but otherwise great moving from ID to classes)

File: header.php [hunk 2]
```diff
@@ -16,8 +16,8 @@
...
       <h1>Setupshop</h1>
       <?php wp_nav_menu( array( 'theme_location' => 'header-menu' ) ); ?>
-    </section>
+    </header>
```
Comment (royscheeren) [inline header.php:23]: Make sure theres a newline at the end of each file.

File: index.php [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
+if ( have_posts() ) : 
+?>
+
+  <ul>
+    <?php while ( have_posts() ) : the_post(); ?>
+	  <li>
+	    <?php the_content(); ?>
+	  </li>
+    <?php endwhile; ?>
+  </ul>
```
Comment (royscheeren) [inline index.php:13]: Looking good ;)

File: style.css [hunk 4]
```diff
@@ -14,34 +14,28 @@ html {
...
-   -webkit-column-rule-style: solid;
-   -webkit-column-rule-width: 1px;
-   -webkit-column-rule-color: #000;
-   -webkit-column-rule: 1px solid #000;*/
 }
 
-section #header {
-   background-color: #fff;
-   margin-bottom: 4rem 0;
+header .site-header {
```
Comment (royscheeren) [inline style.css]: Don't need to specify `header` here. `.site-header` should suffice

File: style.css [hunk 5]
```diff
@@ -14,34 +14,28 @@ html {
...
+   background-color: #000;
+   opacity: .5px;
```
Comment (royscheeren) [inline style.css]: Opacity doesn't do pixels :) `0.5` should do it.

File: style.css [hunk 6]
```diff
@@ -14,34 +14,28 @@ html {
...
+   position: fixed;
 }
 
-h1 {
-   font-family: 'Over the Rainbow';
-   color: #1E90FF;
-   float: left;
+.site-header h1 {
+   float: left !important;
```
Comment (royscheeren) [inline style.css:30]: Don't use important unless you really really really have to. The fact that you need to here means there's something off. You should look into getting those out.

File: style.css [hunk 7]
```diff
@@ -51,26 +45,27 @@ a {
...
    text-decoration: none;
    float: right;
-   /*clear: left;    clear the left of menu, can't find the wp class/id of main content   */
+   clear: left;
    display: inline;
    font-size: 1.5rem;
-   margin-bottom: 10rem; /* wrong way... should simply follow order */
-   height: 4rem !important;
+   margin-bottom: 5rem;
+   height: 0.5rem !important;
```
Comment (royscheeren) [inline style.css]: Again the !importants

File: style.css [hunk 8]
```diff
@@ -102,36 +96,69 @@ a {
...
    padding: 2rem;
-   width: auto;
    color: #000;
+}*/
+
+.read-more-state {
+  display: none;
+}
+
+span.read-more-target {
```
Comment (royscheeren) [inline style.css:142]: You shouldn't need to specify the `span`.

File: style.css [hunk 9]
```diff
@@ -102,36 +96,69 @@ a {
...
+  opacity: 0;
+  max-height: 0;
+  font-size: 0;
+  transition: .25s ease;
+}
+
+.read-more-state:checked ~ .read-more-wrap span.read-more-target {
```
Comment (royscheeren) [inline style.css:149]: Do you know what the `~` does? Is it really necessary in this case because it seems overly complicated.

File: style.css [hunk 10]
```diff
@@ -102,36 +96,69 @@ a {
...
+
+.read-more-state ~ .read-more-trigger:before {
+  content: 'Show more';
+}
+
+.read-more-state:checked ~ .read-more-trigger:before {
+  content: 'Show less';
+}
+
+label.read-more-trigger {
```
Comment (royscheeren) [inline style.css:163]: Shouldn't need to specify the `label`

File: functions.php [hunk 11]
```diff
@@ -55,3 +55,4 @@ function wpdocs_hack_wp_title_for_home( $title )
   return $title;
 }
 ?>
+-->
```
Comment (royscheeren) [inline functions.php:58]: Newline at the end of the file

File: functions.php [hunk 12]
```diff
@@ -38,9 +38,9 @@ function register_my_menu() {
 add_action( 'init', 'register_my_menu' );
 ?>
 
-<?php
+<!--<?php
```
Comment (royscheeren) [inline functions.php:41]: you're putting php in html comment tags, that's not going to work. Check http://www.tizag.com/phpT/comment.php

File: footer.php [hunk 13]
```diff
@@ -1,9 +1,9 @@
 
       <?php wp_footer(); ?>
 
-    <footer>
+    <!--<footer>
       <p>This is our Footer, Here!</p>	
-    </footer>
+    </footer>-->
```
Comment (royscheeren) [inline footer.php:6]: Always make sure that wp_footer comes right before `</body>`

**Review by royscheeren — CHANGES_REQUESTED** [review]
Few comments, nothing major but will help with the overall structure of your site. But yeah the merge conflicts do need to be solved.

File: header.php (hunk 2 above)
Comment (81jumpstr) [reply header.php:23]: I thought I was watching that closely...I guess with more exercise over time it'll soon become a habbit

File: index.php (hunk 3 above)
Comment (81jumpstr) [reply index.php:13]: :)

File: style.css (hunk 4 above)
Comment (81jumpstr) [reply style.css]: Initially I didn't but it wasn't responding and when I included header it fired off. I was also a bit puzzled. I'll take another look at it and let you know

File: style.css (hunk 5 above)
Comment (81jumpstr) [reply style.css]: ...will revisit that

File: style.css (hunk 6 above)
Comment (81jumpstr) [reply style.css:30]: Will do

File: style.css (hunk 7 above)
Comment (81jumpstr) [reply style.css]: Yah, I was concerned to see it starting to look a bit  too much

File: style.css (hunk 8 above)
Comment (81jumpstr) [reply style.css:142]: will revisit

File: style.css (hunk 9 above)
Comment (81jumpstr) [reply style.css:149]: I think it targets all related elements, recently discovered this, I admit I'm not well informed about it. It's one of those I gave a try and wouldn't respond without it

File: style.css (hunk 10 above)
Comment (81jumpstr) [reply style.css:163]: will revisit

File: functions.php (hunk 12 above)
Comment (81jumpstr) [reply functions.php:41]: Ok, will revisit

File: functions.php (hunk 11 above)
Comment (81jumpstr) [reply functions.php:58]: cool

File: footer.php (hunk 13 above)
Comment (81jumpstr) [reply footer.php:6]: Cool

**Comment by 81jumpstr** [comment]
Okay... mostly standard practise related, that sounds like progress, except it was a way around that "something off" you mentioned. Getting good at temporary solutions might help with clients' presentations, buying time while fixing the real issue, as mentioned in the sprint this morning :), but yah, of course it should only be an option with really challenging, technical issues. Thanks a lot, Roy!

**Pull request merged**